src/niscope/metadata/enums.py

```diff
@@ -1079,119 +1069,224 @@
     'VideoSignalFormat': {
         'values': [
             {
-                'name': 'NTSC',
+                'name': 'NISCOPE_VAL_NTSC',
                 'value': 1,
 'documentation': {
 'description': 'NTSC signal format supports line numbers from 1 to 525',
 },
             },
             {
-                'name': 'PAL',
+                'name': 'NISCOPE_VAL_PAL',
                 'value': 2,
 'documentation': {
 'description': 'PAL signal format supports line numbers from 1 to 625',
 },
             },
             {
-                'name': 'SECAM',
+                'name': 'NISCOPE_VAL_SECAM',
                 'value': 3,
 'documentation': {
 'description': 'SECAM signal format supports line numbers from 1 to 625',
 },
             },
             {
                 'name': 'M_PAL',
+                'value': 4,
```

Comment:
All these missing values in one of the source APIs is already reported right?

Reply:
These are already reported.